projects/plugins/jetpack/modules/custom-post-types/portfolios.php

```diff
@@ -821,27 +967,33 @@ static function get_project_class( $portfolio_index_number, $columns ) {
 		 * @param string $class class name of the div.
 		 * @param int $portfolio_index_number iterator count the number of columns up starting from 0.
 		 * @param int $columns number of columns to display the content in.
-		 *
 		 */
-		return apply_filters( 'portfolio-project-post-class', implode( " ", $class ) , $portfolio_index_number, $columns );
+		return apply_filters(
+			'portfolio-project-post-class', // phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores
+			implode( ' ', $class ),
+			$portfolio_index_number,
+			$columns
+		);
 	}
 
 	/**
 	 * Displays the project type that a project belongs to.
 	 *
+	 * @param int $post_id Post ID.
+	 *
 	 * @return html
 	 */
-	static function get_project_type( $post_id ) {
+	private static function get_project_type( $post_id ) {
 		$project_types = get_the_terms( $post_id, self::CUSTOM_TAXONOMY_TYPE );
 
-		// If no types, return empty string
+		// If no types, return empty string.
 		if ( empty( $project_types ) || is_wp_error( $project_types ) ) {
 			return;
 		}
 
-		$html = '<div class="project-types"><span>' . __( 'Types', 'jetpack' ) . ':</span>';
+		$html  = '<div class="project-types"><span>' . __( 'Types:', 'jetpack' ) . '</span>';
```

Comment:
Here and below, I opted to move `:` inside the translatable string since that's much better for translators. In some languages you have a non-breakable space between words and punctuation for example. RTL languages need to be handled too. So this needs to be translated.